targets/playground/Thymio2-natives.cpp

```diff
@@ -131,25 +144,25 @@ static double _fmax(double x, double y)
 
 extern "C" void PlaygroundThymio2Native_leds_top(AsebaVMState *vm)
 {
-	const double r(paramToColor(clampValueTo32(vm->variables[AsebaNativePopArg(vm)])));
-	const double g(paramToColor(clampValueTo32(vm->variables[AsebaNativePopArg(vm)])));
-	const double b(paramToColor(clampValueTo32(vm->variables[AsebaNativePopArg(vm)])));
-	const double a(_fmax(_fmax(r, g), b));
-	
+	const sint16 r(clampValueTo32(vm->variables[AsebaNativePopArg(vm)]));
+	const sint16 g(clampValueTo32(vm->variables[AsebaNativePopArg(vm)]));
+	const sint16 b(clampValueTo32(vm->variables[AsebaNativePopArg(vm)]));
+	const sint16 a(_imax(_imax(r, g), b));
+	const double param(1./_imax(_imax(r, g),_imax(1,b)));
```

Comment:
I do not understand why this computation with a is still necessary. I did it before because I was mixing the colour over the white surface, but now with the texture, cannot we simply call setLedColor() with the r, g, b parameters?